test_install.sh

```diff
@@ -3,4 +3,8 @@
 timeout 20m pip install -vvvv chainer/dist/*.tar.gz --user
 
 # check if cupy is installed
-python -c 'import cupy'
+if [ $CUDNN = none ]; then
```

Comment:
Is `$CUDNN` declared?

How about adding `env['CUDNN'] = conf['cudnn']` to https://github.com/pfnet/chainer-test/pull/178/files#diff-97414e63442fe1e023eeb0b68b38eac5R67